easybuild/easyconfigs/s/Salome/Salome-8.5.0-intel-2020a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Salome'
+version = '8.5.0'
+
+easyblock = "Tarball"
+
+homepage = 'https://www.salome-platform.org/'
+description = """Salome is an open-source software that provides a generic Pre- and Post-Processing platform for numerical
+ simulation. It is based on an open and flexible architecture made of reusable components."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+sources = [{
+    'source_urls': ['https://www.salome-platform.org/downloads/previous-versions/salome-v%(version)s/'],
+    'download_filename':'DownloadDistr?platform=OS1.CO7&version=%(version)s',
+    'filename':'SALOME-%(version)s-CO7-SRC.tgz'
+}]
```

Comment:
@ekieffer better to use this direct download link:
https://files.salome-platform.org/Salome/Salome8.5.0/SALOME-8.5.0-CO7-SRC.tgz